src/main/java/net/openhft/chronicle/bytes/MappedFile.java

```diff
@@ -283,6 +283,7 @@ public void setNewChunkListener(final NewChunkListener listener) {
     @NotNull
     public abstract RandomAccessFile raf();
 
+    @Deprecated
```

Comment:
Should we add comment "To be removed in 2.2n"?

Reply:
The removal of the override is a bit complicated. We need to move from `finalize()` to perhaps using `WeakReference` or something else. The end-user should never invoke this method so I do not think we have to say when.

Reply:
In other words, it is not a part of the public API.